$NetBSD: patch-testsuite_tests_tool-ocamldoc-man_Makefile,v 1.1.2.2 2017/07/24 16:27:15 spz Exp $

After one of our other local patches, output lands in a different
file than what this make job expects.  Follow suit.

--- testsuite/tests/tool-ocamldoc-man/Makefile.orig	2016-11-04 16:08:24.000000000 +0000
+++ testsuite/tests/tool-ocamldoc-man/Makefile
@@ -39,7 +39,7 @@ run: *.mli
 	  F="`basename $$file .mli`"; \
 	  $(OCAMLDOC) $(DOCFLAGS) -hide-warnings -man $ \
 	              -o index $$file; \
-	  tail -n +2 $$F.3o > $$F.result; \
+	  tail -n +2 $$F.3 > $$F.result; \
 	  $(DIFF) $$F.reference $$F.result >/dev/null \
 	  && echo " => passed" || echo " => failed"; \
 	done
@@ -49,6 +49,6 @@ promote: defaultpromote
 
 .PHONY: clean
 clean: defaultclean
-	@rm -f *.result *.html *.tex *.log *.out *.sty *.toc *.css *.aux *.3o
+	@rm -f *.result *.html *.tex *.log *.out *.sty *.toc *.css *.aux *.3
 
 include $(BASEDIR)/makefiles/Makefile.common
